### PR TITLE
Laravel update

### PR DESCRIPTION
**Pull request by bereket-WMDE**
Bug [T284314](https://phabricator.wikimedia.org/T284314)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "superbalist/laravel-google-cloud-storage": "^2.2"
     },
     "require-dev": {
-        "barryvdh/laravel-ide-helper": "^2.10",
+        "barryvdh/laravel-ide-helper": ">=2.8.0 <2.9.2",
```
Comment (addshore) [inline composer.json:32]: This change will also require a new / updated composer.lock to be committedd

File: .phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
...\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #3":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #4":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #5":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #6":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #7":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #8":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #9":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #10":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #11":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #12":0.01,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testOptionsRequestResponds200":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testPostRequestWhenUnauthenticatedRespondes401":0.014}}
```
Comment (addshore) [inline .phpunit.result.cache]: We don't want to commit this file (and should probably add it to the .gitignore)

**Review by addshore — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (bereket-WMDE) [reply composer.json:32]: done

**Comment by addshore** [comment]
Just tried this out locally and didnt find anything missing with the core functionality etc, it all seems to work! :)

**Review by addshore — APPROVED** [review]
LGTM

**Pull request merged**